e2e/tests/upgrades/upgrade_test.go

```diff
@@ -59,8 +62,12 @@ func (s *UpgradeTestSuite) UpgradeChain(ctx context.Context, chain *cosmos.Cosmo
 		Info:   fmt.Sprintf("upgrade version test from %s to %s", currentVersion, upgradeVersion),
 	}
 
-	upgradeProposal := upgradetypes.NewSoftwareUpgradeProposal(fmt.Sprintf("upgrade from %s to %s", currentVersion, upgradeVersion), "upgrade chain E2E test", plan)
-	s.ExecuteAndPassGovV1Beta1Proposal(ctx, chain, wallet, upgradeProposal)
```

Comment:
curious when it's possible to completely switch over and remove this

Reply:
That would be once v7 reaches end of life, right?

Reply:
I think so, but I'm basing it just off one of the feature releases matrices in values.go